tests/test-schema.js

```diff
@@ -0,0 +1,142 @@
+var assert = require('chai').assert;
```

Comment:
I struggle with js testing sometimes. I only used chai because I that's all I knew from other projects